src/js/components/formatDateAndTime.js

```diff
@@ -0,0 +1,9 @@
+export const formatDateAndTime = (dateString) => {
```

Comment:
This file looks like a utility function and not a component. A component is more of UI than anything. Why don't you move it out to another folder such as `utils`.